### PR TITLE
[W-10728932] add draft content for new home page

### PR DESCRIPTION
**Pull request by mojavelinux**

### CODE REVIEW

**Comment by mojavelinux** [comment]
The next step is to fill in the xrefs with real targets in the documentation site. Feel free to push updates to this PR with those updates.

**Comment by mojavelinux** [comment]
We will also need high resolution images for the feature images used on this new page. The current images were extracted from the Figma design comp.

**Comment by cheungaryk** [comment]
ref: [W-10728932](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000qVs4eYAC/view)

Looks good to me, no spelling errors. I tagged our designer (I don't know him Github username) on the GUS ticket above to review the design.

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,86 +1,130 @@
...
 
-* xref:design-center::index.adoc[Design Center]
-* xref:2.x@api-manager::latest-overview-concept.adoc[API Manager]
-* xref:access-management::index.adoc[Access Management]
+* xref:#[CloudHub]
+* xref:#[Connectors]
+* xref:#[DataWeave Language]
+* xref:#[Design Center]
+* xref:#[Exchange]
+* xref:#[Mule Runtime 4.3]
```
Comment (dukesphere) [inline modules/ROOT/pages/index.adoc]: Not sure if this is a placeholder, but I suspect we don't want to point people to 4.3 when 4.4 is the latest even if 4.3 is currently more popular at the moment. A company goal is to get people using the latest, and we're moving away from versioning in the future. 

Same very likely goes for the Studio versions.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,86 +1,130 @@
...
-| xref:3.9@mule-runtime::anypoint-connectors.adoc[Connectors for Mule 3] | xref:connectors::index.adoc[Connectors for Mule 4]
-| xref:3.9@mule-runtime::dataweave.adoc[DataWeave 1 for Mule 3]  | xref:mule-runtime::dataweave.adoc[DataWeave 2 for Mule 4]
-| xref:3.x@apikit::index.adoc[APIkit 3] | xref:4.x@apikit::overview-4.adoc[APIkit 4]
-| xref:3.9@connector-devkit::index.adoc[Connector DevKit] | xref:1.1@mule-sdk::index.adoc[Mule SDKs]
-|===
+Create and publish new API specifications from scratch.
+
+* xref:#[Create a reusable API specification]
+* xref:#[Plan for access]
+* xref:#[Plan for security]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]
* xref:access-management::index.adoc[Plan for access]
* xref:general::security.adoc[Plan for security]
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,86 +1,130 @@
...
+[discrete]
+=== Develop & Test
+
+Build and test APIs and integrations.
 
-== Test Integrations
+* xref:#[Develop your API]
+* xref:#[Connect your data, systems, and apps]
+* xref:#[Develop the Mule app that integrates systems, services, APIs, and devices]
+* xref:#[Build automated tests for your APIs and integrations]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* xref:studio::index.adoc[Develop your API]
* xref:connectors::index.adoc[Connect your data, systems, and apps]
* xref:mule-runtime::mule-app-dev.adoc[Develop the Mule app that integrates systems, services, APIs, and devices]
* xref:dataweave::index.adoc[Access and transform data within the Mule app]
* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,86 +1,130 @@
...
+[discrete]
+=== Deploy
 
-== Deploy Integrations
+Deploy APIs and apps to your hosting targets, and secure them with policies.
 
-Deploy your integration into a production environment.
+* xref:#[Deployment options]
+* xref:#[Configure a proxy for your API]
+* xref:#[Create policies for your API]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* xref:runtime-manager::deployment-strategies.adoc[Deployment options]
* xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]
* xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]
```

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,86 +1,130 @@
...
 
-Once your integration is running in production, you can monitor its performance.
+Monitor your APIs and integrations using dashboards, metrics, and visualization.
 
-* xref:monitoring::index.adoc[Anypoint Monitoring]
-* xref:api-functional-monitoring::index.adoc[API Functional Monitoring]
-* xref:runtime-manager::monitoring.adoc[Runtime Monitoring]
+* xref:#[Monitor Mule app performance]
+* xref:#[Monitor deployed API performance]
+* xref:#[Monitor deployed Mule app performance]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]
```

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,86 +1,130 @@
...
-* https://developer.mulesoft.com/tutorials-and-howtos/getting-started/hello-mule[MuleSoft developer tutorial "Getting Started with MuleSoft"^]
-* https://training.mulesoft.com/[Training^]
+xref:#[View all release notes,role=view-all]
 
-== Archived Documentation
+[#more-resources]
+=== More Resources
 
-When an older version of a product is no longer supported, including products with end-of-life status, the documentation moves to an https://docs.archive.mulesoft.com/[archive site^].
+* xref:#[MuleSoft Help Center]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* https://help.mulesoft.com/s/[MuleSoft Help Center^]
```

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -1,86 +1,130 @@
...
+* xref:#[Developer Resources]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* https://developer.mulesoft.com/tutorials-and-howtos/getting-started/hello-mule[Developer Tutorials^]
```

File: modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -1,86 +1,130 @@
...
+* xref:#[MuleSoft Training]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* https://training.mulesoft.com/[Training^]
```

File: modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -1,86 +1,130 @@
...
+* xref:#[MuleSoft Basics on Trailhead]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* https://trailhead.salesforce.com/content/learn/modules/mulesoft-basics[Learn the Basics of MuleSoft^]
```

File: modules/ROOT/pages/index.adoc [hunk 10]
```diff
@@ -1,86 +1,130 @@
...
-* xref:design-center::index.adoc[Design Center]
-* xref:2.x@api-manager::latest-overview-concept.adoc[API Manager]
-* xref:access-management::index.adoc[Access Management]
+* xref:#[CloudHub]
+* xref:#[Connectors]
+* xref:#[DataWeave Language]
+* xref:#[Design Center]
+* xref:#[Exchange]
+* xref:#[Mule Runtime 4.3]
+* xref:#[Studio 7.4]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
* xref:dataweave::dw-core-functions-filter.adoc[Dataweave Function: filter]
* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
* xref:dataweave::dw-operators.adoc[DataWeave Operators]
* xref:connectors::index.adoc[Connectors]
* xref:mule-runtime::mule-error-concept[Mule Errors]
* xref:mule-runtime::secure-configuration-properties.adoc[Secure Configuration Properties]
* xref:dataweave::dw-core-functions-map.adoc[Dataweave Function: map]
```

File: modules/ROOT/pages/index.adoc [hunk 11]
```diff
@@ -1,86 +1,130 @@
...
+|xref:#[Mule runtime engine for CloudHub]
+
+|June 1
+|xref:#[API Designer 2.17.2]
+
+|June 1
+|xref:#[MUnit Studio Plugin 2.5.1a]
+
+|May 30
+|xref:#[API Designer 2.17.0a]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc]:
```suggestion
|April 11
|xref:api-functional-monitoring/api-functional-monitoring-release-notes.adoc[API Functional Monitoring 2.1]

|April 11
|xref:connector/connector-db.adoc#1-13-0[Database Connector 1.13]

|April 2
|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]

|April 2
|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]

|March 31
|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
```

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (mojavelinux) [reply modules/ROOT/pages/index.adoc]: This text came from the Figma design, so I think it's safe to say it was a placeholder. To solve it, I would recommend dropping the mention of the version. Once set up, the URL will resolve to the latest version.

**Comment by mojavelinux** [comment]
Thanks for looking up and filling in those targets @Cristian-Venticinque!

File: modules/ROOT/pages/index.adoc [hunk 12]
```diff
@@ -1,86 +1,131 @@
...
 
-== Archived Documentation
+[#popular-topics]
+=== Popular Topics
+
+* xref:dataweave::dw-core-functions-filter.adoc[Dataweave Function: filter]
+* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
+* xref:dataweave::dw-operators.adoc[DataWeave Operators]
+* xref:connectors::index.adoc[Connectors]
+* xref:mule-runtime::mule-error-concept[Mule Errors]
```
Comment (dukesphere) [inline modules/ROOT/pages/index.adoc:98]:
```suggestion
* xref:mule-runtime::mule-error-concept.adoc[Mule Errors]
```

File: modules/ROOT/pages/index.adoc [hunk 13]
```diff
@@ -1,86 +1,131 @@
...
+|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]
+
+|April 2
+|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]
+
+|March 31
+|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
+|===
+
+xref:#[View all release notes,role=view-all]
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/index.adoc:123]:
```suggestion
xref:release-notes::quick-refs/by-date-index.adoc[Release Notes by Month,role=view-all]
```

**Comment by cheungaryk** [comment]
Closing this one. The writers wanted to be able to make changes by themselves and they can't do that easily with this PR. I have created a duplicate PR #120 which they are now using actively. Thank you!

**Pull request closed without merging**